nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -10,11 +10,17 @@
 import numpy as np
 import nibabel as nb
```

Comment:
several imports in here appear to not follow the "basic->more specific/elaborate" convention...